examples/analytics_specs/README.md

```diff
@@ -0,0 +1,46 @@
+# atlas-basic - cluster with analytics specs
+
+This example shows how the atlas-basic module can be used to create a cluster defining analytic nodes.
```

Comment:
shows how you can use the atlas-basic module to create a cluster with analytics nodes

Here and elsewhere: analytics, not analytic